scripts/get-sampleapp-url.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eE
+
+KUBECONFIG=""
+DEBUGFILE="/tmp/getsampleappurl.log"
+if [[ -e "${DEBUGFILE}" ]]; then
+    rm -f "${DEBUGFILE}"
+fi
+
+# jq reads from stdin
+function parse_input() {
+  eval "$(jq -r '@sh "export KUBECONFIG=\(.KUBECONFIG) APPNAMESPACE=\(.APPNAMESPACE) APPNAME=\(.APPNAME)"')"
+}
+
+parse_input
+
+if [[ -z "${KUBECONFIG}" ]] || [[ -z "${APPNAMESPACE}" ]] || [[ -z "${APPNAME}" ]]; then
+    echo "[ERROR] one or more input parameter is empty" >> "${DEBUGFILE}"
+    echo "[ERROR] one or more input parameter is empty" >&2
+    SAMPLEAPPROUTE="ERROR"
+else
+    # shellcheck disable=SC2129
+    echo "[INFO] using KUBECONFIG ${KUBECONFIG}" >> "${DEBUGFILE}"
+    # shellcheck disable=SC2129
+    echo "[INFO] using APPNAMESPACE ${APPNAMESPACE}" >> "${DEBUGFILE}"
+    # shellcheck disable=SC2129
+    echo "[INFO] using APPNAME ${APPNAME}" >> "${DEBUGFILE}"
+
+    SAMPLEAPPROUTE="$(oc get routes "${APPNAME}" -n "${APPNAMESPACE}" --no-headers | awk '{print $2}')"
```

Comment:
anything cleaner than using awk? Such as checking something in the json output?

Reply:
pretty same of https://github.com/terraform-ibm-modules/terraform-ibm-websphere-liberty-operator/pull/3#discussion_r1483154302: if I need to parse the json, I would need to use jq or as alternative to move to python